shopify/utils/queries/subscribe.ts

```diff
@@ -0,0 +1,35 @@
+import { gql } from "../../../utils/graphql.ts";
+
+export const query = gql`
+mutation customerCreate($input: CustomerInput!) {
+  payload: customerCreate(input: $input) {
```

Comment:
The API for subscribing a customer on Shopify is called `customerCreate`?

Reply:
Kind of that. 

They use the same table of Customers to do E-mail campaigns, the thing is that some customers accepted e-mail:
![image](https://github.com/deco-cx/apps/assets/76620866/8979a3af-4f21-4d33-9687-65e9ba108b6f)
And each customer is tagged with specific campaigns:
![image](https://github.com/deco-cx/apps/assets/76620866/0922ae96-387d-42b3-ac54-d4bff0b1a28d)

But, you made me realize that I am not dealing with people that already are in the table. Now I will.